resources/shellHelperFunctions.sh

```diff
@@ -137,8 +137,15 @@ write_download_info() {
 
   FILE_EXTENSION="${BASE_FILE##*.}"
 
-  FILE_SIZE=$(/usr/bin/stat -f"%z" "${BASE_PATH}/${BASE_FILE}")
-  MD5_HASH=$(md5 -q "${BASE_PATH}/${BASE_FILE}")
+  # stat flags to get filesize in bytes
+  if [ "$BUILDER_OS" == "mac" ] && [[ $(which stat) == /opt/homebrew/opt/coreutils/libexec/gnubin/stat ]]; then
```

Comment:
This needs to be testing /usr/bin/stat because homebrew stat can live in different places depending on CPU architecture. And isn't this test the wrong way around? `-f%z` is for /usr/bin/stat and `-c%s` is for coreutils stat.

```suggestion
  if [ "$BUILDER_OS" == "mac" ] && [[ $(which stat) == /usr/bin/stat ]]; then
```

Reply:
/me confused

On my win and linux envs, (usr/bin/stat), I have to use flags `-c%s`.
If I try `-f%z`, I get the error 
> stat: unknown option -- %

So I had assumed the original script using `/usr/bin/stat -f"%z"` was coreutils

Reply:
/usr/bin/stat on win and linux are GNU stat (coreutils). /usr/bin/stat on mac is BSD. `-f'%z'` is for BSD stat.

Reply:
ah thanks - I'll include in comments